website/addons/base/views.py

```diff
@@ -40,15 +40,52 @@
 # import so that associated listener is instantiated and gets emails
 from website.notifications.events.files import FileEvent  # noqa
 
-FILE_GONE_ERROR_MESSAGE = u'''
+# TODO - make distiction between filenodes that had GUIDs and were trashed
+# through the OSF/WB and those that were trashed due to not being found at the
+# provider. Currently both cases go through 'FILE_GONE_HAD_GUID'.
+ERROR_MESSAGES = {'FILE_GONE': u'''
 <style>
 .file-download{{display: none;}}
```

Comment:
What are `.file-download`, `.file-share`, and `.file-delete` used for?  If they're obsolete, please delete them.